recipes-core/packagegroups/packagegroup-ni-ptest-smoke.bb

```diff
@@ -31,6 +31,7 @@ RDEPENDS:${PN}:append = "\
 	nettle-ptest \
 	ni-hw-scripts-ptest \
 	ni-test-boot-time-ptest \
+	nilrt-snac-ptest \
```

Comment:
Would these tests fail when snac configuration is not enabled or do they only check that files required to enable it are installed on the system? If the former, I don't think the ptest should be added here.

Reply:
The integration tests do not change the system state and pass regardless of whether or not the system is in SNAC configuration. They are just verifying the correctness of the package installation and that the non-destructive high-level workflows (like --help, --version, and verify) function.